src/main/java/com/github/forax/umldoc/core/Method.java

```diff
@@ -1,45 +1,49 @@
 package com.github.forax.umldoc.core;
 
-import static java.util.Objects.requireNonNull;
```

Comment:
Your changing this class for no good reason, please revert that change.
Please stay focus on what you want to achieve and not on side things

Reply:
I think I have bad config with my checkstyle, it told me that static import were not allowed

Reply:
yep !

Reply:
you should use the CheckStyle config file from the repositiory